backend/LexBoxApi/GraphQL/ProjectMutations.cs

```diff
@@ -73,14 +75,26 @@ public record CreateProjectResponse(Guid? Id, CreateProjectResult Result);
     {
         permissionService.AssertCanManageProject(input.ProjectId);
         var project = await dbContext.Projects.FindAsync(input.ProjectId);
-        if (project is null) throw new NotFoundException("Project not found");
-        var user = await dbContext.Users.FindByEmailOrUsername(input.UserEmail);
-        if (user is null)
+        // We don't use NotFoundException here because that's reserved for username not found further down
+        // Furthermore, username not found will happen often, while project not found can only happen if
+        // one admin deleted a project while another admin was submitting the form. Very rare occurrence.
```

Comment:
It's not an admin only feature
```suggestion
        // a user deleted a project while another user was submitting the form. Very rare occurrence.
```